Doc/library/test.rst

```diff
@@ -287,6 +287,49 @@ The :mod:`test.support` module defines the following constants:
    Set to a filename containing the :data:`FS_NONASCII` character.
 
 
+.. data:: LOOPBACK_TIMEOUT
+
+   Timeout in seconds for tests using a network server listening on the network
+   local loopback interface like ``127.0.0.1``). The timeout is long enough to
+   prevent test failure: it takes an account that the client and the server can
+   run in different threads or even different processes.
+
+   The timeout should be long enough for connect(), recv() and send() methods
+   of socket.socket.
```

Comment:
Markup link to `socket.socket`?

Reply:
Perhaps not needed if the methods are linked.